### PR TITLE
Rectangle

### LINKED ISSUE #14: Deleting elements

### LINKED ISSUE #18: creating a rectangle

### PR DESCRIPTION
**Pull request by KirillSerg**
#18 
 + added creation of arbitrary rectangle element

#14 
 + added deleting selected single element by keyPress and by clicking on delete icon in inspector

### CODE REVIEW

File: src/components/Canvas.tsx [hunk 1]
```diff
@@ -42,18 +44,24 @@ const Canvas = () => {
...
       onMouseMove={(e) => handleMouseMove(e)}
       onMouseUp={onMouseUp}
+      onKeyDown={(e) => handleKeyPress(e.key)}
       className="border-4 border-green-600"
       preserveAspectRatio="xMinYMin meet" //for the SVG container to be on the entire screen, while the elements inside kept the proportions and x=0, y=0 viewBox started from the upper left corner
       viewBox="0 0 1920 1080"
       width="100%"
       height="100%"
       xmlns="http://www.w3.org/2000/svg"
+      tabIndex={0}
```
Comment (roman01la) [inline src/components/Canvas.tsx:64]: why do you need tabIndex here?

File: src/store/store.ts [hunk 2]
```diff
@@ -72,22 +87,32 @@ export const onMouseMoveAtom = atom(
         const newY = selectedElement.y + (update.y - selectingArea.startY)
         set(updateElementsAtom, { ...selectedElement, x: newX, y: newY })
       }
+      if (get(isDrawingAtom) && selectedElement?.x && selectedElement.y) {
```
Comment (roman01la) [inline src/store/store.ts:90]: why the first one here is with null check `selectedElement?.x`, but the second one is without `selectedElement.y`?

**Review by roman01la — APPROVED** [review]

File: src/components/Canvas.tsx (hunk 1 above)
Comment (KirillSerg) [reply src/components/Canvas.tsx:64]: to focus on the canvas.
I need it to listen key press when deleting element

File: src/store/store.ts (hunk 2 above)
Comment (KirillSerg) [reply src/store/store.ts:90]: Selected elements can be null. ```selectedElement?.x``` here I check it. So here ```selectedElement.y``` i don't need this check anymore.
in any case, I change this code already in the next PR.

**Pull request merged**